api.md

```diff
@@ -520,9 +521,13 @@ are before the cut-off point, but only one chunk's worth.
 
 #### Required Arguments [](drop_chunks-required-arguments)
 
+Function requires at least one of the following timestamp arguments. When both timestamp arguments
+are used, the function returns the intersection of those two timestamp ranges.
+
 |Name|Description|
 |---|---|
-| `older_than` | Timestamp of cut-off point for data to be dropped, i.e., anything older than this should be removed.|
+| `older_than` | Timestamp of cut-off point where any full chunks older than this timestamp should be removed. |
```

Comment:
Don't love Timestamp here because it can be interval or timestamp. Maybe "Specification of cut-off point where any full chunks older than this timestamp should be removed. (see below)"

Reply:
Understood, that reads fine to me